content/en/blog/_posts/2024-XX-XX-websocket-transition.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
+date: 2024-XX-XX
+slug: websockets-transition
+author: >
+  [Sean Sullivan](https://github.com/seans3) (Google)
+---
+
+With Kubernetes 1.31, PortForward over Websockets is moving to beta.
+RemoteCommand over Websockets previously graduated to beta in v1.30.
```

Comment:
websockets were already supported (modulo known bugs you fixed) but I think the feature is about making it the default in kubectl, right?

Reply:
I have completely re-written this initial sentence, and I think it addresses your feedback and simplifies the sentence. Please let me know what you think.